### PR TITLE
Chelakhov Leonid/hw4

### PR DESCRIPTION
**Pull request by Chelakhovl**

### CODE REVIEW

File: Chelakhovl/HW3/1.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+53.Although never is often better than *right* now.
+54.If the implementation is hard to explain, it's a bad idea.
+55.If the implementation is easy to explain, it may be a good idea.
+56.Namespaces are one honking great idea -- let's do more of those!"""
+
+#Вирішив для початку зберегти слова в змінних
+bet = "better"
+nev = "never"
+is1 = "is"
+
```
Comment (kolyasalubov) [inline Chelakhovl/HW3/1.py:62]: good job

File: Chelakhovl/HW3/2.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#вирішив задачу за допомогою зрізів
+task2 = input("Enter your number in 1000 for 9999:")
+
```
Comment (kolyasalubov) [inline Chelakhovl/HW3/2.py:3]: good job

File: Chelakhovl/HW3/3.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#створюємо дві зміни в яких значення буде число
+
```
Comment (kolyasalubov) [inline Chelakhovl/HW3/3.py:2]: good job

File: Chelakhovl/HW4/HW4.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Celsius = float(input("Enter a temperature in Celsius: "))
```
Comment (kolyasalubov) [inline Chelakhovl/HW4/HW4.py:1]: variables must be named in lowercase
сelsius

File: Chelakhovl/HW4/HW4.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+Fahrenheit = (Celsius * 9/5) + 32
+
```
Comment (kolyasalubov) [inline Chelakhovl/HW4/HW4.py:3]: good job

**Review by kolyasalubov — APPROVED** [review]

File: Chelakhovl/HW3/1.py (hunk 1 above)
Comment (Chelakhovl) [reply Chelakhovl/HW3/1.py:62]: thanks

File: Chelakhovl/HW3/2.py (hunk 2 above)
Comment (Chelakhovl) [reply Chelakhovl/HW3/2.py:3]: thanks

File: Chelakhovl/HW3/3.py (hunk 3 above)
Comment (Chelakhovl) [reply Chelakhovl/HW3/3.py:2]: thanks

File: Chelakhovl/HW4/HW4.py (hunk 4 above)
Comment (Chelakhovl) [reply Chelakhovl/HW4/HW4.py:1]: Thanks for Your attention

File: Chelakhovl/HW4/HW4.py (hunk 5 above)
Comment (Chelakhovl) [reply Chelakhovl/HW4/HW4.py:3]: thank You

**Pull request merged**